sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlJavaUdfTest.java

```diff
@@ -222,13 +224,18 @@ public void testBinaryJavaUdf() {
     String sql =
         String.format(
             "CREATE FUNCTION matches(str STRING, regStr STRING) RETURNS BOOLEAN LANGUAGE java OPTIONS (path='%s'); "
-                + "SELECT matches(\"a\", \"a\"), 'apple'='beta'",
+                + "SELECT matches('a', 'a'), matches('apple', 'beta')",
```

Comment:
- Would the old version of this query succeed now?
- I assume it is okay to remove the old query because we are getting the same coverage with internal compliance tests?

Reply:
The function is `testBinaryJavaUdf`, so I assume it is trying to test binary UDFs. I fixed what I see as a typo and reverted the changes disabling the test. I'm happy to revert this.

Reply:
Currently, this tests what happens when we mix UDFs and builtin operators. We should probably keep this coverage (maybe change `matches` to `helloWorld` for simplicity) and make `testBinaryJavaUdf` a separate test.

Reply:
I agree, this doesn't need to change for this PR so I reverted my changes.